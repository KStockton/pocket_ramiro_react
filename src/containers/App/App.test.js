import React from 'react';
import { shallow } from 'enzyme';
import { App } from './';

describe('App', () => {
  let wrapper, mockSession;
  beforeEach(() => {
    mockSession = {
      user_id: 6,
      name: 'Me',
      role: 'admin'
    };
    wrapper = shallow(<App session={mockSession}/>);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});



