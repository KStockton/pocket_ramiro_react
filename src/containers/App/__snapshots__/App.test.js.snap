// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<main
  className="route-main"
>
  <Connect(Nav) />
  <ResourceType />
  <Switch>
    <Route
      exact={true}
      path="/"
      render={[Function]}
    />
    <Route
      component={[Function]}
      path="/tickets"
    />
    <Route
      component={[Function]}
      path="/resources"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(withRouter(UserLogin))",
          "type": [Function],
        }
      }
      path="/login"
    />
    <Route
      component={[Function]}
      path="/create-user"
    />
    <Route
      component={[Function]}
      path="/resourcetypes"
    />
    <Route
      component={[Function]}
      path="/create-tickets"
    />
    <Route
      component={[Function]}
      path="/create-parts"
    />
    <Route
      component=""
      path="/create-resources"
    />
    <Route
      component={[Function]}
      path="/create-resourcetypes"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</main>
`;
